js/app.js

```diff
@@ -1,8 +1,3 @@
-$('[type="checkbox"]').click(function(){
-    var element = $(this).next();
-    if ($(this).is(':checked')) {
-       element.addClass('strike')
-    } else {
-       element.removeClass('strike')
-    }
-});
+function printFunction() {
+    window.print();
+}
```

Comment:
Here's a polemic on EventListeners vs. Onclick: http://stackoverflow.com/questions/6348494/addeventlistener-vs-onclick
Generally the fine grain control involved with event listeners is going to be a huge advantage.